static/js/self-check.js

```diff
@@ -154,11 +154,15 @@ function loan() {
     		
     	}).fail(function(jqxhr, textStatus, error) {
     		console.log(jqxhr.responseText);
-            console.log(textStatus);
-            console.log(error);
-    		
+        console.log(textStatus);
+        console.log(error);
     		$("#modalheader").text("");
-    		$("#modalheader").append("item not avaiable for loan.<br/><br/>please see the reference desk for more information<br/><br/><input class='modalclose' type='button' value='close' id='barcodeerrorbutton' onclick='javascript:returnToBarcode();'/>");
+    		if (jqxhr.status == 409) {

```

Comment:
By changing the response codes in the Python code, you can use them here to present different messages to the user.